<launch>
  <arg name="robot" default="scitos"/>

  <arg name="rho" default="1.0"/>
  <arg name="run" default="0"/>
  <arg name="scn" default="A"/>
  <arg name="input_views" default="$(find viper)/data/$(arg robot)/views.json"/>
  <arg name="input_file" default="$(find viper)/data/$(arg robot)/plans-$(arg scn)-$(arg rho)-$(arg run).json"/>
  <!-- <arg name="input_file" default="$(find viper)/data/$(arg robot)/plans.json"/> -->
  <arg name="input_file_values" default="$(find viper)/data/$(arg robot)/view_values.json"/>
  <arg name="input_file_costs" default="$(find viper)/data/$(arg robot)/view_costs.json"/>
  <arg name="output_file" default="$(find viper)/data/$(arg robot)/plans-$(arg scn)-$(arg rho)-$(arg run)-VALUES.json"/>
  <!-- <arg name="output_file" default="$(find viper)/data/$(arg robot)/plan_values.json"/>-->

  <include file="$(find viper)/launch/$(arg robot)/_evaluate_plans.launch"/>

  <node name="evaluate_plans" pkg="viper" type="eval_plans.py" output="screen">
    <param name="robot" type="string" value="$(arg robot)"/>
    <param name="input_views" type="string" value="$(arg input_views)"/>
    <param name="input_file" type="string" value="$(arg input_file)"/>
    <param name="input_file_values" type="string" value="$(arg input_file_values)"/>
    <param name="input_file_costs"  type="string" value="$(arg input_file_costs)"/>
    <param name="output_file" type="string" value="$(arg output_file)"/>
  </node>
</launch>